website/cue/reference/components/transforms/geoip.cue

```diff
@@ -183,6 +192,50 @@ components: transforms: geoip: {
 							}
 							groups: ["City"]
 						}
+						country_name: {
+							description: """
+								The country name associated with the IP address. Is looked up using
+								the configured locale.
+								"""
+							required: true
+							type: string: {
+								examples: ["United States", "United Kingdom"]
+							}
+							groups: ["City"]
+						}
```

Comment:
Shouldn't these be nested under `geoip`?

Reply:
<img width="842" alt="image" src="https://user-images.githubusercontent.com/383250/170057796-1da2deed-d26a-41bc-bf9e-3f0b8639591e.png">

Yeah, it appears this is correct, there's just some weird indentation going on in my diff view.